lettuce/flows/couette.py

```diff
@@ -11,20 +11,28 @@
 class CouetteFlow2D(object):
     def __init__(self, resolution, reynolds_number, mach_number, lattice):
         self.resolution = resolution
+        self.shape = (resolution, resolution)
+        self._mask = np.zeros(shape=self.shape, dtype=bool)
         self.units = UnitConversion(
             lattice,
             reynolds_number=reynolds_number, mach_number=mach_number,
             characteristic_length_lu=resolution, characteristic_length_pu=1,
             characteristic_velocity_pu=1
         )
 
-    def analytic_solution(self, x, t=0):
-        # TODO
-        raise NotImplementedError
+    def analytic_solution(self, x):
+        dvdy = 1/self.resolution
+        x, y = self.grid
+        u = np.array([dvdy*y + v0], dtype=float)
+        return u
 
     def initial_solution(self, x):
         return np.array([0 * x[0]], dtype=float), np.array([0 * x[0], 0 * x[1]], dtype=float)
 
+    @property
+    def mask(self):
+        return self._mask\
```

Comment:
```suggestion
        return self._mask
```
or it was correct?

Reply:
It does not change the code's behavior but it's a good suggestion